uproot/rootio.py

```diff
@@ -1452,24 +1499,38 @@ def tostring(self):
         return numpy.asarray(self, dtype=self._dtype).tostring()
 
 class TArrayC(TArray):
+    _classname = b"TArray"
+    classname = "TArray"
```

Comment:
Are these strings meant to be "TArrayC"?

Reply:
Thanks! You're right, and this is the reason why _not_ to write WET code. However, trade-offs.

I did another sweep over all the `classname` assignments that exist and they all look right. The number of non-streamer class definitions won't be growing or won't be growing much. They are exceptions—most classes get defined from streamers.

Oh, I should add `classnames` (both the underscored and non-underscored version) for the classes that define the streamers themselves.